### PR TITLE
Bl 4779 accessibility changes page titles

### PR DESCRIPTION
**Pull request by Steveybrown**
- All Page titles now respect accessibility in terms of page titles

Apologies for number of file changes but changed some files to reflect of style guide I believe. 🤒

### CODE REVIEW

**Comment by Steveybrown** [comment]
Thanks for the suggestion @przemos , much better approach. 

Ready for review folks 🎉

File: webapp/src/main/resources/template/vrm.hbs [hunk 1]
```diff
@@ -37,4 +37,4 @@
     </form>
 
 {{/partial}}
-{{> master}}
+{{> master pageTitle='What is the vehicle’s registration (number plate)? – MOT reminders'}}
```
Comment (paddyfox) [inline webapp/src/main/resources/template/vrm.hbs]: Surround by Double quotes

File: webapp/src/main/resources/template/unsubscribe-confirmation.hbs [hunk 2]
```diff
@@ -19,4 +19,4 @@
         </nav>
     </form>
 {{/partial}}
-{{> master}}
+{{> master pageTitle='You’ve unsubscribed – MOT reminders'}}
```
Comment (paddyfox) [inline webapp/src/main/resources/template/unsubscribe-confirmation.hbs]: Surround by Double quotes

File: webapp/src/main/resources/template/home.hbs [hunk 3]
```diff
@@ -45,4 +45,4 @@
         </div>
     </div>
 {{/partial}}
-{{> master}}
+{{> master pageTitle='Get MOT reminders'}}
```
Comment (paddyfox) [inline webapp/src/main/resources/template/home.hbs]: Surround by Double quotes

File: webapp/src/test/java/uk/gov/dvsa/motr/web/resource/UnsubscribeConfirmedResourceTest.java [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+        assertEquals(UnsubscribeViewModel.class, TEMPLATE_ENGINE_STUB.getContext(Map.class).get("viewModel").getClass());
+        String dataLayerString = (String) TEMPLATE_ENGINE_STUB.getContext(Map.class).get("dataLayer");
+        UnsubscribeViewModel viewModel = (UnsubscribeViewModel) TEMPLATE_ENGINE_STUB.getContext(Map.class).get("viewModel");
+        assertEquals("test@this-is-a-test-123", viewModel.getEmail());
+        assertEquals("10 July 2015", viewModel.getExpiryDate());
+        assertEquals("TEST-VRM", viewModel.getRegistration());
+        assertEquals("{\"vrm\":\"TEST-VRM\"}", dataLayerString);
+    }
+
```
Comment (paddyfox) [inline webapp/src/test/java/uk/gov/dvsa/motr/web/resource/UnsubscribeConfirmedResourceTest.java]: Remove additional line

File: webapp/src/main/java/uk/gov/dvsa/motr/web/mapper/NotFoundExceptionMapper.java [hunk 5]
```diff
@@ -2,6 +2,8 @@
 
 import uk.gov.dvsa.motr.web.render.TemplateEngine;
 
+import java.util.HashMap;
+
```
Comment (Thomas-Franklin) [inline webapp/src/main/java/uk/gov/dvsa/motr/web/mapper/NotFoundExceptionMapper.java]: Seems to be unused

File: webapp/src/main/java/uk/gov/dvsa/motr/web/mapper/UnhandledExceptionMapper.java [hunk 6]
```diff
@@ -5,6 +5,9 @@
 
 import uk.gov.dvsa.motr.web.render.TemplateEngine;
 
+import java.util.HashMap;
+import java.util.StringJoiner;
+
```
Comment (Thomas-Franklin) [inline webapp/src/main/java/uk/gov/dvsa/motr/web/mapper/UnhandledExceptionMapper.java]: Also unused?

**Pull request merged**